tests/test_user_repo_fallback.py

```
@@ -0,0 +1,65 @@
+    RoleMappingConfig,
+    RuntimeConfig,
+    ScoringConfig,
+)
+
+
+def test_user_repo_fallback_on_org_unauthorized(monkeypatch, caplog) -> None:
+    config = BotConfig(
+        runtime=RuntimeConfig(
+            mode="dry-run",
+            log_level="INFO",
+            data_dir="/tmp",
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)